docs/reference/content/guides/annotations.md

```diff
@@ -0,0 +1,292 @@
+    boolean dropDups() default false;
+    boolean background() default false;
+    boolean sparse() default false;
+    int expireAfterSeconds() default -1;
+}
+```
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| value() | The sort direction for the index |
+| name() | The name of the index |
+| unique() | Requires values in the index to be unique |
+| dropDups() | Drop any duplicate values during the creation of a unique index. |
+| background() | Create this index in the background.  There are some [considerations](http://docs.mongodb.org/manual/tutorial/build-indexes-in-the-background/#considerations) to keep in mind.
+| sparse() | Create a [sparse](http://docs.mongodb.org/manual/core/index-sparse/) index |
+| expireAfterSeconds() | Creates a [TTL Index](http://docs.mongodb.org/manual/core/index-ttl/) on a date field. |
+
+## Id
+Marks a field in an `@Entity` to be the "_id" field in mongodb.
+
+## Property
+An optional annotation instructing Morphia to persist field in to the document given to mongodb.  By default, the field name is used as 
+the property name.  This can be overridden by passing a String with the new name to the annotation.
+
+## Transient
+Instructs Morphia to ignore this field when converting an entity to a document.  The java keyword `transient` can also be used instead.
+
+## Serialized
+Instructs Morphia to serialize this field.
+
+```java
+public @interface Serialized {
+  String value() default Mapper.IGNORED_FIELDNAME;
+  boolean disableCompression() default false;
+}
+```
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| value() | the field name to use in the document |
+| disableCompression() | By default, Morphia compresses the `byte[]` after serialization.  Setting this to true disables the compression. |
+
+
+## NotSaved
+Instructs Morphia to ignore this field when saving but will still be loaded.
+ 
+_Good for data migration._
+
+## AlsoLoad
+When a field gets remapped to a new name, you can either update the database and migrate all the fields at once or use this annotation 
+to tell Morphia what older names to try if the current one fails.  It is an error to have values under both the old and new key names 
+when loading a document.
+
+_Good for data migration._
+
+```java
+public @interface AlsoLoad {
+  String[] value();
+}
+```
+
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| value() | The array of names to try when loading the field |
+
+## Version
+Marks a field in an `@Entity` to control optimistic locking for that entity. If the versions change while modifying an entity (including 
+deletes) a `ConcurrentModificationException` will be thrown. This field will be automatically managed for you -- there is no need to set
+ a value and you should not do so anyway.  If the another name beside the Java field name is desired, a name can be passed to this 
```

Comment:
"If another name".

This could really use an example.  Most people won't know how this actually works based on the description you've provided here.